### DISCUSSION TITLE
Add Profile Activity UI

### DISCUSSION THREAD

**Discussion by ShubhadeepKarmakar** [oppia/oppia-android#4801] [Developer Announcements]
Before :
![Screenshot_2022-12-23-10-19-40-433_org oppia android](https://user-images.githubusercontent.com/99060332/209311425-95e03891-468b-42f9-8407-0369da636970.jpg)
After :
![Screenshot_2022-12-23-10-18-30-571_org oppia android](https://user-images.githubusercontent.com/99060332/209311511-ce609b3b-a80f-42de-a858-2a9342fdf87b.jpg)
I think the 2nd option is better, can I file an issue to change it?

**Comment by MohitGupta121** [comment]
@ShubhadeepKarmakar actually these are all according to [mock](https://xd.adobe.com/view/3dca36c2-5115-419c-b25e-0f10526b077c-6899/screen/c08d51aa-b7c1-4c10-ad9f-41522247ff95/) designed by the desgined team. 
And if you wants to contribute in that team feel free to check [discussion](https://github.com/oppia/oppia-android/discussions/4788#discussion-4656255) and fill contributor form.

If this solved your query then please mark this discussion as resolved by marking this as answer.
Feel Free to [contribute](https://github.com/oppia/oppia-android/discussions/4788#discussion-4656255) to this project.
Thanks

**Comment by BenHenning** [comment]
I created an issue from this discussion here: #4827. An issue seems like a better venue to discuss potential issues with the app.

Thanks for starting this discussion @ShubhadeepKarmakar!

**Reply by ShubhadeepKarmakar** [reply]
> I created an issue from this discussion here: #4827. An issue seems like a better venue to discuss potential issues with the app.
> 
> Thanks for starting this discussion @ShubhadeepKarmakar!

Can I make a pr against the #4827 issue?
